lib/Mojo/Message/Request.pm

```diff
@@ -166,16 +166,23 @@ sub _parse_env {
   my $headers = $self->headers;
   my $url     = $self->url;
   my $base    = $url->base;
+
+  my $port = $env->{SERVER_PORT};
+ 
   for my $name (keys %$env) {
     my $value = $env->{$name};
     next unless $name =~ s/^HTTP_//i;
     $name =~ y/_/-/;
     $headers->header($name => $value);
 
     # Host/Port
-    $value =~ s/:(\d+)$// ? $base->host($value)->port($1) : $base->host($value)
-      if $name eq 'HOST';
+    if ( $name eq 'HOST' ) {
```

Comment:
The code is not run through perltidy. Please do so:

```
perltidy -b lib/Mojo/Message/Request.pm
perltidy -b t/mojo/request_cgi.t
```

Reply:
Fixed, sent a new copy up